Assets/MRTK/Tests/PlayModeTests/TestProfiles/TestCustomPointerMediatorInputProfile.asset

```diff
@@ -35,7 +35,7 @@ MonoBehaviour:
     handedness: 7
     pointerPrefab: {fileID: 1247086986094436, guid: d5b94136462644c9873bb3347169ae7e,
       type: 3}
-  - controllerType: 47
+  - controllerType: 1071
```

Comment:
Is there any way to make this easily optional, or is the recommendation to remove "articulated hand" from the list of sources the teleport pointer is valid for?

Reply:
I think we want the teleport pointer to be available and enabled for all articulated hands was my current impression. Do we want to make the teleport pointer a more "opt in" feature? There's no mechanisms to enable/disable it beyond the profiles themselves, though the teleport gesture will work/display anything unless the teleport system itself is enabled.